cmd/download.go

```diff
@@ -82,8 +143,9 @@ func init() {
 	downloadCmd.Flags().Bool("generate-imageset", false, "Generate imageset.yaml only")
 	downloadCmd.Flags().Bool("du-profile", false, "Pre-cache telco 5G DU operators")
 	downloadCmd.Flags().Bool("skip-imageset", false, "Skip imageset.yaml generation")
-	downloadCmd.Flags().StringP("hub-version", "", "", "RHACM operator version, in X.Y.Z format")
-	downloadCmd.MarkFlagRequired("hub-version")
+	downloadCmd.Flags().StringP("hub-version", "", "", "(deprecated) RHACM operator version, in X.Y.Z format")
+	downloadCmd.Flags().StringP("acm-version", "", "", "Advanced Cluster Management operator version, in X.Y.Z format")
+	downloadCmd.Flags().StringP("mce-version", "", "", "MultiCluster Engine operator version, in X.Y.Z format")
```

Comment:
mce-version must be set to required: downloadCmd.MarkFlagRequired("mce-version")

We use the mce-version to obtain the ai-images without requiring the --ai-img flag

Reply:
This would break existing usage with just --hub-version, hence the extra checks during arg validation